lib/resources/r4/charge_item.yaml

```diff
@@ -105,3 +105,17 @@ fields:
         - display: Bill Code Schedule
           system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/14002
           info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-14002-bill-code-schedule
+
+  - name: performer
```

Comment:
should we be more specific and call it performer.function? I know we have done that in other places like [Patient ](http://fhir.cerner.com/millennium/r4/base/individuals/patient/#terminology-bindings) for patient.identifier.type

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/682/commits/be52c43e6f896a3e8f843c8d310a90bd9b638d09#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR109

Reply:
thanks for making that change.